tests/test_prior_external.py

```diff
@@ -8,25 +8,25 @@
 
 The test prior is a gaussian half-ring, combined with a gaussian in one of the tests.
 
-For manual testing, and observing/plotting the density, pass `manual=True` to `body of test`.
+For manual testing, and observing/plotting the density, pass `manual=True`
+to `body of test`.
 """
 # Local
-from cobaya.conventions import _prior
 from .common_external import info_string, info_callable, info_mixed, info_import
 from .common_external import body_of_test
 
 
 def test_prior_external_string(tmpdir):
-    body_of_test(info_string, _prior, tmpdir)
+    body_of_test(info_string, "prior", tmpdir)
```

Comment:
Should be grabbed from conventions, and next ones too

Reply:
There are hundreds of these..